server/auth/google/google.go

```diff
@@ -109,7 +111,11 @@ func (c *Config) StartSession(state string) *auth.Session {
 
 // Exchange authorizes the session and returns an access token.
 func (c *Config) Exchange(code string) (*oauth2.Token, error) {
-	return c.config.Exchange(oauth2.NoContext, code)
+	t, err := c.config.Exchange(oauth2.NoContext, code)
+	if err == nil {
+		metrics.M.AuthExchange.WithLabelValues("google").Inc()
```

Comment:
It's OK, I wrote it at 1am so questions are appreciated! I could bump all this up to `provider.go` but thought capturing the module would be good in case there's some future version where multiple providers are supported.